less/components/querybuilder.less

```diff
@@ -543,6 +757,55 @@ div.column-container {
             margin-left: 15px;
         }
     }
+
+    .sort-item {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        min-height: 40px;
+        max-width: 100%;
+
+        .path-parts {
+            display: flex;
+            flex-wrap: wrap;
+
+            .part {
+                padding: 0 2px;
```

Comment:
Can we discuss the benefits of using `@spacer` when you're back? There are 40 lines that use margin/padding with px, so it will take me a long time to convert them without breaking alignment (since spacer is 1em, which changes depending on context, meaning I have to use the devtools and my very weary eyes to judge whether it's equivalent, for every single case).